### DISCUSSION TITLE
File .env

### DISCUSSION THREAD

**Discussion by amorepsiche** [iisgiua/giuaschool#566] [Richieste di aiuto] [answered]
Salve, ho installato Giua sul mio database altervista, tuttavia ho notato che digitando https://nome_mio_sito/.nev il contenuto è pubblico e tutti possono leggere i file che non dovrebbero essere pubblici. Come faccio a rendere inaccessibili i suddetti file? Grazie!

**Comment by trinko** [comment, marked as answer]
Nell'installazione di giuaschool sono presenti due file che impediscono la visualizzazione del file **.env**. 
Per cui penso che la tua installazione non sia stata eseguita correttamente o che, dopo l'installazione, tu abbia modificato qualche impostazione dal pannello di controllo di Altervista.

In ogni caso, puoi caricare nuovamente i seguenti file e controllare che la visualizzazione di **.env** sia bloccata.
I due file che proteggono il sistema su siti come Altervista sono:
- **.htaccess**
- **public/.htaccess**

**_Attenzione_**: non caricarli direttamente dal gestore di file di Altervista perché potrebbe non funzionare. Usa un software come Filezilla per caricare i file.

**Comment by amorepsiche** [comment]
Ho provveduto a installare nuovamente tutti file. Grazie per la risposta.


Il giorno mar 29 ago 2023 alle 15:29 Antonello Dessì <
***@***.***> ha scritto:
